docs/faq.md

```diff
@@ -0,0 +1,36 @@
+# KEDA-HTTP Frequently Asked Questions
+
+## How is this Project Similar or Different from [Osiris](https://github.com/deislabs/osiris)?
+
+Osiris and KEDA-HTTP have similar features:
+
+- Autoscaling, including scale-to-zero of compute workloads
+- Native integration to the [Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+
+However, Osiris and KEDA-HTTP differ in several ways:
+
+- Autoscaling concerns are implemented separately from the application resources - `Service`, `Ingress`, `Deployment` and more in KEDA-HTTP. With Osiris, those concerns are baked into each app resource.
+- The KEDA-HTTP operator can automatically deploy and configure networking and compute resources necessary for an HTTP application to autoscale. Osiris does not have this functionality.
+- Osiris is currently archived in GitHub
+
+## How is this Project Similar or Different from [KNative](https://knative.dev/)?
+
+KNative serving and KEDA-HTTP both have core support for autoscaling, including scale-to-zero of compute workloads. KEDA-HTTP is focused solely on deploying production-grade autoscaling HTTP applications, while KNative builds in additional functionality:
+
+- Pure [event-based workloads](https://knative.dev/docs/eventing/). [KEDA core](https://github.com/kedacore/keda), without KEDA-HTTP, can support such workloads natively. If you have a more advanced use case than KEDA core can support, [Dapr](https://dapr.io/) may be a good choice for you.
```

Comment:
Not sure if we should refer to Dapr here as it's not really related?
```suggestion
- Pure [event-based workloads](https://knative.dev/docs/eventing/). [KEDA core](https://github.com/kedacore/keda), without KEDA-HTTP, can support such workloads natively.
```

Reply:
Works for me 👍